packages/js/src/ui/config/default-appearance.ts

```diff
@@ -0,0 +1,13 @@
+import { Variables } from '../context';
+
+export const defaultVariables: Required<Variables> = {
+  colorPrimary: '#0081F1',
+  colorPrimaryForeground: 'white',
+  colorSecondary: '#F3F3F3',
+  colorSecondaryForeground: '#1A1523',
```

Comment:
I am not a big fan of this after all as it doesn't tell me immediately what to do. Text on primary might be better after all.